### PR TITLE
feat(pds-ember): add Copyright and GlobalFooter

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: PDS-226

- Add `<Pds::Copyright>`
- Add `<Pds::GlobalFooter>`
- Update `<Pds::App>` story to demo usage of `<Pds::GlobalFooter>`
- Update App docs with cross links to relevant components

### Copyright
Copyright - Basic

### GlobalFooter
__Basic__
GlobalFooter - Basic

__Copyright + Product Name__
GlobalFooter - Copyright + Product Name

__Copyright + Nav__
GlobalFooter - Copyright + Nav

__Copyright + Product Name + Nav__
GlobalFooter - Copyright + Product Name + Nav

### CODE REVIEW

File: packages/pds-ember/addon/components/pds/global-footer/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component'
+
+/**
+ * @class PdsGlobalFooter
+ */
+export default class PdsGlobalFooter extends Component {
+  get year() {
+    let year = (new Date()).getFullYear()
+    return year
+  }
```
Comment (CITguy) [inline packages/pds-ember/addon/components/pds/global-footer/index.js:10]: remove getter (no longer needed)

File: packages/pds-ember/addon/components/pds/copyright/index.hbs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div
+  class="pds-copyright"
+  ...attributes
+  data-test-copyright
+>
+  <Pds::Logomark::Hashicorp
+    data-test-copyright-logomark
+  />
+
+  <span data-test-copyright-text>
```
Comment (CITguy) [inline packages/pds-ember/addon/components/pds/copyright/index.hbs:10]: either make use of the data attr or remove it

File: packages/pds-ember/app/styles/pds/components/global-footer/__config.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    > * + * {
+      margin-inline-start: theme.$size--xs;
+    }
+  }
+
+  &__productName,
+  &__nav {
+    align-items: center;
+    display: flex;
+    line-height: 1;
```
Comment (CITguy) [inline packages/pds-ember/app/styles/pds/components/global-footer/__config.scss:41]: clean up `line-height` application

File: packages/pds-ember/tests/integration/components/copyright/index-test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { module, test } from 'qunit'
+import { setupRenderingTest } from 'ember-qunit'
+import { render } from '@ember/test-helpers'
+import { hbs } from 'ember-cli-htmlbars'
+
+const ROOT = '[data-test-copyright]'
+
```
Comment (CITguy) [inline packages/pds-ember/tests/integration/components/copyright/index-test.js:7]: Update to test individual pieces of the component (logomark and text)

**Review by CITguy — COMMENTED** [review]
see comments for changes

File: packages/pds-ember/addon/components/pds/global-footer/index.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import Component from '@glimmer/component'
+
+/**
+ * @class PdsGlobalFooter
+ */
+export default class PdsGlobalFooter extends Component {}
```
Comment (lackeyjb) [inline packages/pds-ember/addon/components/pds/global-footer/index.js:6]: When a component class is empty this you don't technically need to have the file. Is there a reason we add these in structure and cloud-ui @meirish?

File: packages/pds-ember/app/styles/pds/components/copyright/__config.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+@use "../logomark/_config" as Logomark;
+
+@mixin layout {
+  /* [debug] #{$_module}@layout */
+  align-items: center;
+  display: inline-flex;
+  line-height: 1;
+
+  // child spacing
+  > * + * {
```
Comment (lackeyjb) [inline packages/pds-ember/app/styles/pds/components/copyright/__config.scss:16]: thanks for adding the comment here!

**Review by lackeyjb — APPROVED** [review]
Looks good 👍. I had one small comment about the necessity of some of the empty JS component files, but it was more of a nitpick/question about why we're creating them

File: packages/pds-ember/addon/components/pds/global-footer/index.js (hunk 5 above)
Comment (CITguy) [reply packages/pds-ember/addon/components/pds/global-footer/index.js:6]: They aren't absolutely necessary, but there have been strange quirks with Storybook integration when they're not present (specifically with the YUIDoc comments).  Additionally, it just removes an extra step later on if/when we need to add behavior to a component.

**Pull request merged**